src/databases/ADIOS2/ADIOS2.xml

```diff
@@ -31,6 +31,7 @@
     </LIBS>
     <LIBS components="EPar">
       ${ADIOS2_PAR_LIB}
+      ${ADIOS2_LIB}
```

Comment:
I think it would be better to have FindADIOS2.cmake set  up the PAR_LIB to also include the serial versions, as is done for Windows.  As it stands, this change will create duplications on Windows for the link line.

Reply:
> As it stands, this change will create duplications on Windows for the link line.

And...

Reply:
@biagas have a look at 697cbe8c0aa47830e055d38b6d91677423d9bcfc. It is untested.